602/documentation framework (#681)
guardicore/monkey: docs/content/usage/getting-started.md

---
title: "Getting Started"
date: 2020-05-26T21:01:12+03:00
draft: false
weight: 1
pre: "<i class='fas fa-play-circle'></i> "
tags: ["usage"]
---

## Using the Infection Monkey

After deploying the Monkey Island in your environment, navigate to `https://<server-ip>:5000`.

### First-time setup

On your first login, you'll be asked to set up a username and password for the Monkey Island server. [See this page for more details](../accounts-and-security).

### Run the Monkey

To get the Infection Monkey running as fast as possible, click **Run Monkey**. Optionally, you can configure the Monkey before you continue by clicking **Configuration** (see [how to configure the monkey](../configuration)).

To run the monkey, select one of the following options:

![Run Page](/images/usage/getting-started/run_page_with_arrows.jpg "Run Page")

> 1. Click **Run on C&C Server** to run the Infection Monkey on the Monkey Island server. This simulates an attacker trying to propagate from a machine in the Monkey Island subnet.

Reviewer: ```suggestion
1. Click **Run on C&C Server** to run the Infection Monkey on the Monkey Island server. This simulates an attacker trying to propagate through local network from Monkey Island machine.
```

Author: Done
